atomsci/ddm/pipeline/perf_data.py

```diff
@@ -1279,16 +1238,12 @@ def get_pred_values(self):
         """
         ids = sorted(self.pred_vals.keys())
```

Comment:
unrelated but didn't we agree not to sort predictions?

Reply:
I double checked, KFoldRegressionPerfData and KFoldClassificationPerfData sort in `get_pred_values()`, `get_real_values()` and `get_weights()` while SimpleClassificationPerfData, SimpleRegressionPerfData and SimpleHybridPerfData do NOT sort by ID.

Reply:
I think the order that the keys are returned from self.pred_vals is totally random. So it's either sorted or totally random. Not sure which would be better.